lib/cn/cn_tree.c

```diff
@@ -1534,48 +1569,68 @@ cn_comp_update_split(
     }
 }
 
+void
+cn_comp_kvset_append(
+    struct cn_compaction_work *work,
+    struct cn_tree_node       *node,
+    struct kvset              *kvset)
+{
```

Comment:
I'd name this `cn_comp_update_subspill()` to match `cn_comp_update_split()` and `_kvcompact()` since they all perform the same function, which is to update the tree with the new kvsets.